src/components/NFTGem/Wrapper.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="relative w-full">
-    <slot />
+    <slot :gemLevel="gemLevel" />
```

Comment:
Prefer using kebab case for props, ref: https://vuejs.org/guide/components/props.html#prop-name-casing
```suggestion
    <slot :gem-level="gemLevel" />
```